### PR TITLE
This is the code for the HW7 Code Reviews

### PR DESCRIPTION
**Pull request by reedan88**
Hey everyone, this is the code I put up for the code reviews. Please use this for the HW#7.
Ummm....why don't I have any comments?

### CODE REVIEW

**Review by mguerrap — COMMENTED** [review]
comment example

File: CodeReview/Andrew_code.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import os
+import numpy as np
+import pandas as pd
+import xarray as xr
+import matplotlib.pyplot as plt
+
```
Comment (abtadams) [inline CodeReview/Andrew_code.py:6]: #Create functions to load station data
#Function to load Neah Bay station data

File: CodeReview/Andrew_code.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
```
Comment (abtadams) [inline CodeReview/Andrew_code.py:7]: Do you need to define datadir?
I can't run this function

File: CodeReview/Andrew_code.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+def load_Neah_Bay(datadir):
+    """
+    Function to load the Neah Bay tidal station data from 2015 - 2016
+    and returns a dataframe and a Datetime Index object
+    Datadir is the directory path to where the data is located
+    """
```
Comment (abtadams) [inline CodeReview/Andrew_code.py:13]: Returns none if files are not located in specified directory

**Review by abtadams — COMMENTED** [review]
Nice work

**Pull request closed without merging**